add missing build command

bertux opened the pull request:
without building first it can't find the dist directory from where it tries to start

bertux committed:
add missing build command

bertux committed:
title link fix

bertux committed:
MONGODB_URI env display fix

bertux committed:
add env sample

leonprou commented on apps/ensemble-engine/src/main.ts:
```diff
@@ -1,9 +1,23 @@
 
 async function bootstrap() {
   console.log('Starting Ensemble Engine...');
+  console.log('process.env.MONGODB_URI:', process.env.MONGODB_URI);
```
Need to move those prints to the services where they are used/loaded. This way it's messy

bertux replied:
So let's just remove them first.
It's not needed to make it work and it's already warned for RPC when missing and printed when it's not missing.
So another PR should be done to manage the MongoDB URI the same way than RPC URL.

leonprou reviewed (changes requested):
Thanks for the fixes!
Please see my comment here - https://github.com/ensemble-codes/ensemble-engine/pull/4/files#r1764865122

bertux committed:
Cleaning debug logs

Checking URI of MongoDB should be done like URL of RPC.
Should keep main.ts as clear as possible.

bertux commented:
> Thanks for the fixes! Please see my comment here - https://github.com/ensemble-codes/ensemble-engine/pull/4/files#r1764865122

I agree with you, I added the logging where the dotconfig was imported but it's only needed for the debug logging so it's better to remove it, that's what I did.
Now it's waiting for your final review and merge.

leonprou reviewed (approved):
LGTM!
